src/library/rekordbox/rekordboxfeature.cpp

```diff
@@ -43,6 +43,26 @@ const QString kPdbPath = QStringLiteral("PIONEER/rekordbox/export.pdb");
 const QString kPLaylistPathDelimiter = QStringLiteral("-->");
 const double kLongestPosition = 999999999.0;
 
+enum TrackColor {
+    TRACK_COLOR_PINK = 1,
+    TRACK_COLOR_RED,
+    TRACK_COLOR_ORANGE,
+    TRACK_COLOR_YELLOW,
+    TRACK_COLOR_GREEN,
+    TRACK_COLOR_AQUA,
+    TRACK_COLOR_BLUE,
+    TRACK_COLOR_PURPLE
+};
+
+constexpr mixxx::RgbColor::optional_t kPinkTrackColor = mixxx::RgbColor::optional(0xF870F8);
```

Comment:
These constants are never null and don't need to use the optional type. I suggest to use mixxx::RgbColor directly, should be assignment compatible.